app/models/katello/system.rb

```diff
@@ -69,16 +62,18 @@ class System < Katello::Model
     before_update :update_foreman_host, :if => proc { |r| r.environment_id_changed? || r.content_view_id_changed? }
 
     scope :in_environment, ->(env) { where('environment_id = ?', env) unless env.nil? }
-    scope :completer_scope, ->(options) { readable(options[:organization_id]) }
```

Comment:
If I understand correctly, it's a scoped search method http://www.rubydoc.info/github/wvanbergen/scoped_search/master/ScopedSearch%2FAutoCompleteBuilder%3Acompleter_scope that wasn't even working because readable doesn't take an argument iirc. I don't see it used elsewhere in the code, so I removed it.